run-tests.sh

```diff
@@ -48,18 +48,18 @@ fi
 ADDON_LIST=()
 while IFS=  read -r -d $'\0'; do
     ADDON_LIST+=("$REPLY")
-done < <(find "${KUBEADDONS_REPO}/addons" -mindepth 2 -maxdepth 2 -type d -print0)
+done < <(find "${KUBEADDONS_REPO}/addons" -maxdepth 2 -type d -print0)
```

Comment:
What's this for? I thought we were requiring that depth for a reason.

Reply:
addresesd via #13 the tests didn't run before